examples/automm/object_detection/finetune_coco_format.py

```diff
@@ -0,0 +1,167 @@
+"""
+The example to finetune an object detection model in AutoMM in COCO format.
+
+An example to finetune an MMDetection model on COCO:
+    python finetune_coco_format.py \
+        --train_path coco17/annotations/instances_train2017.json \
+        --test_path coco17/annotations/instances_val2017.json \
+        --checkpoint_name yolov3_mobilenetv2_320_300e_coco \
+        --lr <learning_rate> \
+        --epochs <epochs>
+
+An example to finetune an MMDetection model on VOC:
+    First, use this script to convert the VOC dataset to COCO format:
+    https://github.com/open-mmlab/mmdetection/blob/9d3e162459590eee4cfc891218dfbb5878378842/tools/dataset_converters/pascal_voc.py
+    Then, run:
+    python finetune_coco_format.py \
+        --train_path /media/data/datasets/voc/VOCdevkit/VOCCOCO/voc07_trainval.json \
```

Comment:
Use relative path or placeholder

Reply:
Will change this after Yongxing's PR is merged. There are some VOC related changes in that PR.